CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -555,59 +641,65 @@ Roaming range:		20m from network mote.
 Size:				1.2m x 1.2m x 2.4m
 Disrupt DC:			30
 Pwn DC:				70
-Effect:				The titan multipurpose drone is a bipedal megalith of death
+
+    The titan multipurpose drone is a bipedal megalith of death
 and science. Plasma induction core explodes violently when destroyed. 
 The titan comes with a 7.62 repeater that has a 20% chance to miss and deals 55 
 damage. fed from a 150 round drum. May also make melee attacks with its offhand.
-20% chance to miss. Deals 160 damage, kinda like being punched by a backhoe. 
-When a drone is reclaimed, it frees up the control point cost that it occupied,
-but does not reimburse its controller's Nanites that were used to assemble it.
+20% chance to miss. Deals 160 damage, kinda like being punched by a backhoe.
```

Comment:
These drones have not been completely converted to combat 2.0 with APLs or d10 accuracy.

Reply:
blocker. Will change.

Reply:
When you fix this, I'll make sure to review again. (I see you asked for a 2nd review)